src/index.ts

```diff
@@ -432,15 +432,14 @@ const clearKey = async (ctx: Context): Promise<string[]> => {
 	const toDeleteArray = [...toDelete];
 
 	if (toDeleteArray.length > 0) {
-		ctx.wshimLogger.log(`\nKeys cleared: ${toDeleteArray.join('\n')}`);
-	} else {
-		ctx.wshimLogger.log('\nNo keys were cleared.');
-	}
-
-	while (toDeleteArray.length > 0) {
-		await ctx.bucket.ISSUANCE_KEYS.delete(toDeleteArray.splice(0, 1000));
+		await ctx.bucket.ISSUANCE_KEYS.delete(toDeleteArray);
+		ctx.waitUntil(clearDirectoryCache(ctx));
 	}
```

Comment:
This deletes my fix, you can't delete more than 1000 keys, we don't expect to ever do that though, so this was only "technically" a fix